packages/sdk/src/evm/contracts/index.ts

```diff
@@ -198,6 +214,9 @@ export const NFTDropInitializer = {
   getAbi: async () =>
     (await import("@thirdweb-dev/contracts-js/dist/abis/DropERC721.json"))
       .default,
+  getV3Abi: async () =>
```

Comment:
How come its `V3` for `DropERC721` but `V2` for the others

Reply:
following the version of the contract